src/apps/project-settings/ProjectSettings.css

```diff
@@ -34,4 +54,70 @@
   display: flex;
   gap: 0.2em;
   align-items: center;
-} 
\ No newline at end of file
+}
```

Comment:
We should definitely move those general styles (for the switch) into the global stylesheet to avoid repetition further down the line.